docs/source/AdministratorGuide/Systems/DataManagement/dfc.rst

```diff
@@ -0,0 +1,80 @@
+.. _dfc:
+
+------------------
+Dirac File Catalog
+------------------
+
+The DIRAC File Catalog (DFC) is a full replica and metadata catalog integrated to DIRAC. It has a very modulare structure, allowing for several backends. The interaction with the backend is handled by `Managers` in such a way that the interface exposed to the users remains always the same.
+
+There are two main sets of managers:
+* the historical ones, offering the full range of functionalities and used by most VO
+* and the LHCb ones, where a subsets of the functionalities related to user defined metadata are not tested, but optimized for scalling and consistency.
```

Comment:
"LHCb ones"... only? Can't they be used by other VOs?

Also "scalling" -> "scaling"

Reply:
Only us as far as I know, but nothing prevents any VO to use it, providing they are ok with untested features, knowing that they might never appear if it impacts the scaling+consistency we chose :)